src/io/file.js

```diff
@@ -2,7 +2,41 @@
 // http://github.com/p-v-o-s/infragram-js.
 
 module.exports = function File(options, processor) {
-
+  let getMouseEvent = () => {
```

Comment:
OK, i tested it in GitPod and got this error:

```
Uncaught ReferenceError: getMouseEvent is not defined
    at Object.downloadImage (infragram.js:551:30)
    at Object.download (infragram.js:141:22)
    at HTMLButtonElement.onclick (8080-somyasinghal-infragram-vtxgv1qu3w5.ws-us43.gitpod.io/:163:107)
```

But I looked and I believe you have to recompile the `/dist/` files one more time, and/or add them to the pull request with git - your most recent change isn't appearing in `/dist/infragram.js`. Does that make sense? Thank you and sorry again for the delay!!!!

Reply:
Sorry, @jywarren for the late response as I was busy with my exams. I have recompiled the `/dist/` files and added them to the pull request. I have also tested on my system and found image is getting downloaded. Could you please review it once?